### PR TITLE
Remove all occurences of {{Draft|draft}} in translated content

### PR DESCRIPTION
**Pull request by tannerdolby**
Related to [#10723](https://github.com/mdn/content/discussions/10273)

Translated content had 312 combined occurrences of either `{{Draft}}` or `{{draft}}` in 309 files. This PR removes all of those occurrences.

### CODE REVIEW

File: files/pt-br/web/api/document_object_model/how_to_create_a_dom_tree/index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 translation_of: Web/API/Document_object_model/How_to_create_a_DOM_tree
 original_slug: DOM/Referencia_do_DOM/How_to_create_a_DOM_tree
 ---
-<p>{{draft}}</p>
+<p></p>
```
Comment (josielrocha) [inline files/pt-br/web/api/document_object_model/how_to_create_a_dom_tree/index.html:7]:
```suggestion
```

File: files/pt-br/web/api/push_api/best_practices/index.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
 translation_of: Web/API/Push_API/Best_Practices
 original_slug: Web/API/API_Push/Best_Practices
 ---
-<p>{{draft}}</p>
+<p></p>
```
Comment (josielrocha) [inline files/pt-br/web/api/push_api/best_practices/index.html:14]:
```suggestion
```

File: files/pt-br/web/performance/critical_rendering_path/index.html [hunk 3]
```diff
@@ -4,7 +4,7 @@
 translation_of: Web/Performance/Critical_rendering_path
 original_slug: Web/Performance/caminho_de_renderizacao_critico
 ---
-<p>{{draft}}</p>
+<p></p>
```
Comment (josielrocha) [inline files/pt-br/web/performance/critical_rendering_path/index.html:7]:
```suggestion
```

File: files/ru/learn/performance/business_case_for_performance/index.html [hunk 4]
```diff
@@ -7,7 +7,7 @@
 
 <div>{{PreviousMenu("Learn/Performance/Mobile", "Learn/Performance")}}</div>
 
-<p>{{draft}}</p>
+<p></p>
```
Comment (josielrocha) [inline files/ru/learn/performance/business_case_for_performance/index.html:10]:
```suggestion
```

File: files/ru/mozilla/index.html [hunk 5]
```diff
@@ -9,7 +9,7 @@
   - Расширенный
 translation_of: Mozilla
 ---
-<div>{{draft}}</div>
+<div></div>
```
Comment (josielrocha) [inline files/ru/mozilla/index.html:12]:
```suggestion
```

File: files/ru/web/api/element/createshadowroot/index.html [hunk 6]
```diff
@@ -3,7 +3,7 @@
 slug: Web/API/Element/createShadowRoot
 translation_of: Web/API/Element/createShadowRoot
 ---
-<p>{{draft}}</p>
+<p></p>
```
Comment (josielrocha) [inline files/ru/web/api/element/createshadowroot/index.html:6]:
```suggestion
```

File: files/ru/web/api/validitystate/toolong/index.html [hunk 7]
```diff
@@ -10,7 +10,7 @@
   - Справочник
 browser-compat: api.ValidityState.tooLong
 ---
-<div>{{draft}}</div>
+<div></div>
```
Comment (josielrocha) [inline files/ru/web/api/validitystate/toolong/index.html:13]:
```suggestion
```

File: files/de/mozilla/index.html [hunk 8]
```diff
@@ -6,7 +6,7 @@
 translation_of: Mozilla
 ---
 <div>
```
Comment (josielrocha) [inline files/de/mozilla/index.html:8]:
```suggestion
```

File: files/de/mozilla/index.html [hunk 9]
```diff
@@ -6,7 +6,7 @@
...
-  {{draft}}</div>
+  </div>
```
Comment (josielrocha) [inline files/de/mozilla/index.html:9]:
```suggestion
```

File: files/de/web/api/document/registerelement/index.html [hunk 10]
```diff
@@ -14,7 +14,7 @@
 <p><strong>Warnung: </strong>document.registerElement() läuft zu Gunsten von <a href="/en-US/docs/Web/API/CustomElementRegistry/define">customElements.define()</a> aus.</p>
 </div>
 
-<p>{{draft}}</p>
+<p></p>
```
Comment (josielrocha) [inline files/de/web/api/document/registerelement/index.html:17]:
```suggestion
```

File: files/de/web/css/tools/linear-gradient_generator/index.html [hunk 11]
```diff
@@ -3321,7 +3321,7 @@ <h3 id="JavaScript_Content">JavaScript Content</h3>
 				for (var i=0; i</pre>
 </div>
 
-<p>{{Draft}}</p>
+<p></p>
```
Comment (josielrocha) [inline files/de/web/css/tools/linear-gradient_generator/index.html:3324]:
```suggestion
```

**Comment by josielrocha** [comment]
I made some suggestions to pt-br files, but there are similar code snippets in other laguages too. What do you think?

**Comment by SphinxKnight** [comment]
+1 with @josielrocha, if the macro is the only content of an element (`<p>` or `<div>`), the element itself should be removed.

**Comment by tannerdolby** [comment]
> +1 with @josielrocha, if the macro is the only content of an element (<p> or <div>), the element itself should be removed.

I completely agree, thank you for the review @josielrocha! 

There is no point to leave the empty `<p>` or `<div>` elements lying around. Mine as well remove those too if all they had as

File: files/pt-br/web/api/document_object_model/how_to_create_a_dom_tree/index.html (hunk 1 above)
Comment (tannerdolby) [reply files/pt-br/web/api/document_object_model/how_to_create_a_dom_tree/index.html:7]: I will remove that empty tag, thank you!

File: files/pt-br/web/api/push_api/best_practices/index.html (hunk 2 above)
Comment (tannerdolby) [reply files/pt-br/web/api/push_api/best_practices/index.html:14]: ditto

File: files/pt-br/web/performance/critical_rendering_path/index.html (hunk 3 above)
Comment (tannerdolby) [reply files/pt-br/web/performance/critical_rendering_path/index.html:7]: ditto

File: files/fr/conflicting/web/api/document_object_model_656f0e51418b39c498011268be9b3a10/index.html [hunk 12]
```diff
@@ -10,7 +10,7 @@
 translation_of_original: Web/Guide/API/DOM
 original_slug: Web/Guide/DOM
 ---
-<p>{{draft}}</p>
+<p></p>
```
Comment (SphinxKnight) [inline files/fr/conflicting/web/api/document_object_model_656f0e51418b39c498011268be9b3a10/index.html:13]:
```suggestion

```

File: files/fr/learn/html/cheatsheet/index.html [hunk 13]
```diff
@@ -9,7 +9,7 @@
 translation_of: Learn/HTML/Cheatsheet
 original_slug: Apprendre/HTML/Cheatsheet
 ---
-<div>{{draft}}</div>
+<div></div>
```
Comment (SphinxKnight) [inline files/fr/learn/html/cheatsheet/index.html:12]:
```suggestion

```

File: files/fr/mozilla/add-ons/webextensions/api/userscripts/working_with_userscripts/index.html [hunk 14]
```diff
@@ -10,7 +10,7 @@
 translation_of: Mozilla/Add-ons/WebExtensions/API/userScripts/Working_with_userScripts
 original_slug: Mozilla/Add-ons/WebExtensions/API/userScripts/travailler_avec_userScripts
 ---
-<p>{{draft}}</p>
+<p></p>
```
Comment (SphinxKnight) [inline files/fr/mozilla/add-ons/webextensions/api/userscripts/working_with_userscripts/index.html:13]:
```suggestion

```

File: files/fr/orphaned/web/css/tools/cubic_bezier_generator/index.html [hunk 15]
```diff
@@ -7,7 +7,7 @@
 translation_of: Web/CSS/Tools/Cubic_Bezier_Generator
 original_slug: Web/CSS/Tools/Cubic_Bezier_Generator
 ---
-<p>{{Draft}}</p>
+<p></p>
```
Comment (SphinxKnight) [inline files/fr/orphaned/web/css/tools/cubic_bezier_generator/index.html:10]:
```suggestion

```

File: files/fr/tools/remote_debugging/debugging_firefox_desktop/index.html [hunk 16]
```diff
@@ -8,7 +8,7 @@
 translation_of: Tools/Remote_Debugging/Debugging_Firefox_Desktop
 original_slug: Outils/Débogage_distant/Debugging_Firefox_Desktop
 ---
-<p>{{draft}}</p>
+<p></p>
```
Comment (SphinxKnight) [inline files/fr/tools/remote_debugging/debugging_firefox_desktop/index.html:11]:
```suggestion

```

File: files/fr/web/api/document/registerelement/index.md [hunk 17]
```diff
@@ -14,7 +14,7 @@ translation_of: Web/API/Document/registerElement
 
 > **Attention :** document.registerElement() est déprécié en faveur de [customElements.define()](/fr/docs/Web/API/CustomElementRegistry/define).
 
-{{draft}}
+
```
Comment (SphinxKnight) [inline files/fr/web/api/document/registerelement/index.md:17]:
```suggestion
```

File: files/fr/web/api/document/tooltipnode/index.md [hunk 18]
```diff
@@ -11,7 +11,7 @@ translation_of: Web/API/Document/tooltipNode
 ---
 {{ ApiRef("DOM") }}
 
-{{draft}}
+
```
Comment (SphinxKnight) [inline files/fr/web/api/document/tooltipnode/index.md:14]:
```suggestion
```

File: files/fr/web/css/tools/linear-gradient_generator/index.md [hunk 19]
```diff
@@ -7,6 +7,6 @@ tags:
 translation_of: Web/CSS/Tools/Linear-gradient_Generator
 original_slug: Web/CSS/Outils/Générateur_de_dégradés_linéaires
 ---
-{{Draft}}
+
```
Comment (SphinxKnight) [inline files/fr/web/css/tools/linear-gradient_generator/index.md:10]:
```suggestion
```

File: files/fr/web/javascript/the_performance_hazards_of_prototype_mutation/index.md [hunk 20]
```diff
@@ -7,7 +7,7 @@ tags:
 translation_of: Web/JavaScript/The_performance_hazards_of__[[Prototype]]_mutation
 original_slug: Web/JavaScript/Performance_les_dangers_liés_à_la_modification_de_Prototype
 ---
-{{draft}}
+
```
Comment (SphinxKnight) [inline files/fr/web/javascript/the_performance_hazards_of_prototype_mutation/index.md:10]:
```suggestion
```

File: files/fr/web/media/index.md [hunk 21]
```diff
@@ -3,7 +3,7 @@ title: Technologies media Web
 slug: Web/Media
 translation_of: Web/Media
 ---
-{{draft}}
+
```
Comment (SphinxKnight) [inline files/fr/web/media/index.md:6]:
```suggestion
```

File: files/fr/web/performance/performance_budgets/index.md [hunk 22]
```diff
@@ -9,7 +9,7 @@ tags:
 translation_of: Web/Performance/Performance_budgets
 original_slug: Web/Performance/Budgets_de_performance
 ---
-{{draft}}
+
```
Comment (SphinxKnight) [inline files/fr/web/performance/performance_budgets/index.md:12]:
```suggestion
```

File: files/fr/web/progressive_web_apps/index.html [hunk 23]
```diff
@@ -12,7 +12,7 @@
   - Web app
 translation_of: Web/Progressive_web_apps
 ---
-<p>{{draft}}</p>
+<p></p>
```
Comment (SphinxKnight) [inline files/fr/web/progressive_web_apps/index.html:15]:
```suggestion
```

File: files/fr/web/svg/attribute/presentation/index.md [hunk 24]
```diff
@@ -3,7 +3,7 @@ title: Attributs SVG de présentation
 slug: Web/SVG/Attribute/Presentation
 translation_of: Web/SVG/Attribute/Presentation
 ---
-{{draft}}
+
```
Comment (SphinxKnight) [inline files/fr/web/svg/attribute/presentation/index.md:6]:
```suggestion
```

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
I've made a couple suggestions regarding empty elements (or lines).

**Comment by tannerdolby** [comment]
@SphinxKnight Thank you for reviewing :) 

I opened a separate PR in #3090 that removes all empty `<p>` or `<div>` elements that aren't being used. Should I still commit the proposed changes here as they will be fixed in #3090 when its merged?

File: files/fr/conflicting/web/api/document_object_model_656f0e51418b39c498011268be9b3a10/index.html (hunk 12 above)
Comment (tannerdolby) [reply files/fr/conflicting/web/api/document_object_model_656f0e51418b39c498011268be9b3a10/index.html:13]: I removed all of these in #3090, I'm happy to remove it on this branch too if that is needed.

File: files/fr/web/javascript/the_performance_hazards_of_prototype_mutation/index.md (hunk 20 above)
Comment (tannerdolby) [reply files/fr/web/javascript/the_performance_hazards_of_prototype_mutation/index.md:10]: this was not addressed in my other PR so I will commit this suggested change, thanks!

File: files/fr/web/media/index.md (hunk 21 above)
Comment (tannerdolby) [reply files/fr/web/media/index.md:6]: ditto

**Comment by SphinxKnight** [comment]
> Should I still commit your proposed changes from above on this branch as they will be fixed in #3090 when its merged?
I would prefer this PR not to introduce "new" empty elements (so there's no need to chase them afterwards). That's not a strong blocker for me but the "cost" of preventing this here does not seem too high to me.

That being written, thanks a lot for #3090 and #3093 :)

File: files/ja/mdn/structures/page_types/api_method_subpage_template/index.html [hunk 25]
```diff
@@ -26,7 +26,7 @@ <h3 id="Top_macros" name="Top_macros">先頭で使用するマクロ</h3>
 
 <ul>
  <li>\{{APIRef("<em>GroupDataName</em>")}} — this generates the left hand reference sidebar showing quick reference links related to the current page. For example, every page in the <a href="/ja/docs/Web/API/WebVR_API">WebVR API</a> has the same sidebar, which points to the other pages in the API. To generate the correct sidebar for your API, you need to add a GroupData entry to our KumaScript GitHub repo, and include the entry's name inside the macro call in place of <em>GroupDataName</em>. See our <a href="/ja/docs/MDN/Contribute/Structures/API_references/API_reference_sidebars">API reference sidebars</a> guide for information on how to do this.</li>
- <li>\{{Draft}} — this generates a <strong>Draft</strong> banner that indicates that the page is not yet complete, and should only be removed when the first draft of the page is completely finished. After it is ready to be published, you can remove this.</li>
+ <li>\ — this generates a <strong>Draft</strong> banner that indicates that the page is not yet complete, and should only be removed when the first draft of the page is completely finished. After it is ready to be published, you can remove this.</li>
```
Comment (mfuji09) [inline files/ja/mdn/structures/page_types/api_method_subpage_template/index.html:29]: This is a description of the `{{Draft}}` macro, so removing the `{{Draft}}` breaks it.
If the `{{Draft}}` macro shouldn't be used, it is necessary to remove entire the paragraph, I think.

File: files/ja/mdn/structures/page_types/api_property_subpage_template/index.html [hunk 26]
```diff
@@ -27,7 +27,7 @@ <h3 id="Top_macros" name="Top_macros">先頭で使用するマクロ</h3>
 
 <ul>
  <li>\{{APIRef("<em>グループデータ名</em>")}} — これは左端にあるサイドバーを生成し、現在のページに関連するクイックリファレンスのリンクを表示します。例えば、 <a href="/ja/docs/Web/API/WebVR_API">WebVR API</a> の中の各ページは同じサイドバーを持ち、この API 内の他のページを指します。 API で正しいサイドバーを生成するには、 KumaScript の GitHub リポジトリにグループデータの項目を追加し、マクロ呼び出しの中で<em>グループデータ名</em>のところに項目名を入れる必要があります。これを行う方法についてのガイドは <a href="/ja/docs/MDN/Contribute/Structures/API_references/API_reference_sidebars">API リファレンスサイドバー</a>を参照してください。</li>
- <li>\{{Draft}} — this generates a <strong>Draft</strong> banner that indicates that the page is not yet complete, and should only be removed when the first draft of the page is completely finished. After it is ready to be published, you can remove this.</li>
+ <li>\ — this generates a <strong>Draft</strong> banner that indicates that the page is not yet complete, and should only be removed when the first draft of the page is completely finished. After it is ready to be published, you can remove this.</li>
```
Comment (mfuji09) [inline files/ja/mdn/structures/page_types/api_property_subpage_template/index.html:30]: This is similar to previous one.

File: files/ja/mdn/structures/page_types/api_reference_page_template/index.html [hunk 27]
```diff
@@ -24,7 +24,7 @@ <h3 id="Top_macros">先頭で使用するマクロ</h3>
 
 <ul>
  <li>\{{APIRef("<em>グループデータ名</em>")}} — これは左端にあるサイドバーを生成し、現在のページに関連するクイックリファレンスのリンクを表示します。例えば、 <a href="/ja/docs/Web/API/WebVR_API">WebVR API</a> の中の各ページは同じサイドバーを持ち、この API 内の他のページを指します。 API で正しいサイドバーを生成するには、 KumaScript の GitHub リポジトリにグループデータの項目を追加し、マクロ呼び出しの中で<em>グループデータ名</em>のところに項目名を入れる必要があります。これを行う方法についてのガイドは <a href="/ja/docs/MDN/Contribute/Howto/Write_an_API_reference/Sidebars">API リファレンスサイドバー</a>を参照してください。</li>
- <li>\{{Draft}} — これは、 <strong>Draft</strong> バナーを生成してページがまだ完成していないことを示すもので、ページの最初の草稿が完全に完成した場合にのみ削除するべきものです。公開できる状態になったら、これを削除することができます。</li>
+ <li>\ — これは、 <strong>Draft</strong> バナーを生成してページがまだ完成していないことを示すもので、ページの最初の草稿が完全に完成した場合にのみ削除するべきものです。公開できる状態になったら、これを削除することができます。</li>
```
Comment (mfuji09) [inline files/ja/mdn/structures/page_types/api_reference_page_template/index.html:27]: It is similar to the previous one.

**Review by mfuji09 — CHANGES_REQUESTED** [review]
I have reviewed 105 of 312 files (ja).
There are three issues in them.

**Review by sashasushko — APPROVED** [review]
Approve for RU-files from @mdn/yari-content-ru

File: files/ja/mdn/structures/page_types/api_method_subpage_template/index.html (hunk 25 above)
Comment (tannerdolby) [reply files/ja/mdn/structures/page_types/api_method_subpage_template/index.html:29]: 👋 @mfuji09 

Thank you for the review. I will address the 3 issues you mentioned.

File: files/ja/mdn/structures/page_types/api_property_subpage_template/index.html (hunk 26 above)
Comment (tannerdolby) [reply files/ja/mdn/structures/page_types/api_property_subpage_template/index.html:30]: ditto

**Comment by tannerdolby** [comment]
Ok great! Thanks @sashasushko

**Review by josielrocha — APPROVED** [review]
Approve for pt-br files @mdn/yari-content-pt-br

**Comment by tannerdolby** [comment]
Thanks @josielrocha!

**Review by irvin — APPROVED** [review]

**Comment by SphinxKnight** [comment]
@tannerdolby I've pushed what was needed to help fix conflict after Markdown conversion here as well.

**Comment by tannerdolby** [comment]
@SphinxKnight Ok great! Thank you.

**Review by yujo11 — APPROVED** [review]

File: files/es/learn/css/css_layout/index.html [hunk 28]
```diff
@@ -18,7 +18,7 @@
   - float
 translation_of: Learn/CSS/CSS_layout
 ---
-<div>{{draft}}</div>
+<div></div>
```
Comment (JuanVqz) [inline files/es/learn/css/css_layout/index.html:21]:
```suggestion

```

File: files/es/mdn/tools/kumascript/index.html [hunk 29]
```diff
@@ -6,7 +6,7 @@
 ---
 <div>{{MDNSidebar}}</div>
 
-<div>{{draft}}</div>
+<div></div>
```
Comment (JuanVqz) [inline files/es/mdn/tools/kumascript/index.html:9]:
```suggestion

```

File: files/es/mozilla/firefox/releases/61/index.html [hunk 30]
```diff
@@ -3,7 +3,7 @@
 slug: Mozilla/Firefox/Releases/61
 translation_of: Mozilla/Firefox/Releases/61
 ---
-<div>{{FirefoxSidebar}}</div><div>{{draft}}</div>
+<div>{{FirefoxSidebar}}</div><div></div>
```
Comment (JuanVqz) [inline files/es/mozilla/firefox/releases/61/index.html:6]:
```suggestion
<div>{{FirefoxSidebar}}</div>
```

File: files/es/web/accessibility/aria/aria_techniques/using_the_aria-label_attribute/index.html [hunk 31]
```diff
@@ -7,7 +7,7 @@
   - agente
 translation_of: Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute
 ---
-<div>{{draft}}</div>
+<div></div>
```
Comment (JuanVqz) [inline files/es/web/accessibility/aria/aria_techniques/using_the_aria-label_attribute/index.html:10]:
```suggestion

```

File: files/es/web/api/xmlhttprequest/responsetext/index.html [hunk 32]
```diff
@@ -3,7 +3,7 @@
 slug: Web/API/XMLHttpRequest/responseText
 translation_of: Web/API/XMLHttpRequest/responseText
 ---
-<div>{{draft}}</div>
+<div></div>
```
Comment (JuanVqz) [inline files/es/web/api/xmlhttprequest/responsetext/index.html:6]:
```suggestion

```

File: files/es/web/performance/index.html [hunk 33]
```diff
@@ -8,7 +8,7 @@
   - Rendimiento Web
 translation_of: Web/Performance
 ---
-<div>{{draft}}</div>
+<div></div>
```
Comment (JuanVqz) [inline files/es/web/performance/index.html:11]:
```suggestion

```

File: files/es/web/progressive_web_apps/developer_guide/index.html [hunk 34]
```diff
@@ -16,7 +16,7 @@
   - progressive
 translation_of: Web/Progressive_web_apps/Developer_guide
 ---
-<p>{{draft}}</p>
+<p></p>
```
Comment (JuanVqz) [inline files/es/web/progressive_web_apps/developer_guide/index.html:19]:
```suggestion

```

File: files/es/web/progressive_web_apps/index.html [hunk 35]
```diff
@@ -10,7 +10,7 @@
   - aplicación
 translation_of: Web/Progressive_web_apps
 ---
-<div>{{draft}}</div>
+<div></div>
```
Comment (JuanVqz) [inline files/es/web/progressive_web_apps/index.html:13]:
```suggestion

```

File: files/es/web/progressive_web_apps/installing/index.html [hunk 36]
```diff
@@ -20,7 +20,7 @@
 translation_of: Web/Progressive_web_apps/Developer_guide/Installing
 original_slug: Web/Progressive_web_apps/Developer_guide/Installing
 ---
-<p>{{draft}}</p>
+<p></p>
```
Comment (JuanVqz) [inline files/es/web/progressive_web_apps/installing/index.html:23]:
```suggestion

```

**Review by JuanVqz — APPROVED** [review]
Removing not necessary `p` and `div` tags for ES.

File: files/ja/web/api/fetch_api/basic_concepts/index.html [hunk 37]
```diff
@@ -0,0 +1,74 @@
+---
```
Comment (irvin) [inline files/ja/web/api/fetch_api/basic_concepts/index.html:1]: after rebase of the PR had added some un-relative files. 

I feel that we may need to separate the PR into each locale, making it to be easier to review and merge separately.

And to ensure it's not been blocked longly and needs of continence rebase, reduce the risk of broken during each rebase.

**Review by irvin — CHANGES_REQUESTED** [review]
@tannerdolby after rebase of the PR had grown some unrelative files.

I feel that we may need to separate the PR into each locale, making it easier to review and merge separately.

And can ensure it's not been blocked longly and the needs of rebasing, reduce the risk of broken during each rebase.

File: files/fr/web/css/list_of_proprietary_css_features/index.md [hunk 38]
```diff
@@ -0,0 +1,151 @@
+---
```
Comment (irvin) [inline files/fr/web/css/list_of_proprietary_css_features/index.md:1]: another example here

File: files/ja/web/api/fetch_api/basic_concepts/index.html (hunk 37 above)
Comment (tannerdolby) [reply files/ja/web/api/fetch_api/basic_concepts/index.html:1]: will update shortly

File: files/fr/web/css/list_of_proprietary_css_features/index.md (hunk 38 above)
Comment (tannerdolby) [reply files/fr/web/css/list_of_proprietary_css_features/index.md:1]: ditto

**Review by irvin — APPROVED** [review]

**Comment by irvin** [comment]
orz... conflict again

**Comment by SphinxKnight** [comment]
> @tannerdolby after rebase of the PR had grown some unrelative files.
> 
> I feel that we may need to separate the PR into each locale, making it easier to review and merge separately.
> 
> And can ensure it's not been blocked longly and the needs of rebasing, reduce the risk of broken during each rebase.

I second @irvin here, this PR is taking too much time for something that should work well with separate PR (see recent work w/ folks from mdn/content on macro removal).

Regarding `fr`, since https://github.com/mdn/yari/pull/6219 has been merged, I'm going to open another PR which will subsume this one. (cc @teoli2003 / @mdn/localization-team-leads )

**Comment by schalkneethling** [comment]
> I second @irvin here, this PR is taking too much time for something that should work well with separate PR (see recent work w/ folks from mdn/content on macro removal).

When the pull request was opened, we were under the impression that this was ready for removal. It turned out not to be the case, so this pull request has been hanging around longer than originally intended. I definitely do not want to lose the work done by Tanner, so if he is happy to resolve the conflicts, let’s give him an opportunity to finish the work. We can then determine the next steps afterward.

Thank you, everyone. I believe we have a more defined process of how we go about these changes and will follow this process going forward.

**Comment by hochan222** [comment]
The PR period is too long(and I think there will be more conflicts like this in the future), so @tannerdolby (or else) needs to resolve the conflicts in batches and merge them or divide them by locale.

There is no clear opinion in the above comment, so it seems that a decision is needed. (cc: @tannerdolby , @schalkneethling )

Conflicts about the Korean locale seem to be okay to merge immediately after resolution. Please let me know if ko-locale needs to create a separate PR.

Thank you.

**Comment by schalkneethling** [comment]
Oh wow yeah, there are a lot of conflicts here. @tannerdolby, it seems all the conflicts are in `fr` and `ja`. I wonder whether it would make sense to refactor this one to remove those changes? We can then have separate pull requests for `fr` and `ja` from the relevant communities. Let me know your thoughts.

**Comment by tannerdolby** [comment]
@schalkneethling Yeah, there are SO many conflicts here. I hadn't merged in a few months so maybe merging with `main` will fix some of them which I can do today/tonight. But anyhow, I do completely agree. This has been a behemoth of a PR to maintain for all parties. I think having separate pull requests for `fr` and `ja` from relevant communities will be a better and the more efficient plan of action. We could then simply refactor/remove from this PR like you mentioned.

**Comment by schalkneethling** [comment]
> @schalkneethling Yeah, there are SO many conflicts here. I hadn't merged in a few months so maybe merging with `main` will fix some of them which I can do today/tonight. But anyhow, I do completely agree. This has been a behemoth of a PR to maintain for all parties. I think having separate pull requests for `fr` and `ja` from relevant communities will be a better and the more efficient plan of action. We could then simply refactor/remove from this PR like you mentioned.

Thanks, @tannerdolby!

**Comment by cw118** [comment]
@tannerdolby `fr`, `zh-CN` and `zh-TW` have already been taken care of while `ja` has an open PR that I believe will be merged soon (cf. #5693), so if we want to keep this PR, I'd say the changes for these locales should simply be removed.

Thank you again for all the work you've done here :)

**Comment by tannerdolby** [comment]
@schalkneethling No problem! I see there is only one test failing now and its in regards to `files/fr/_redirects.txt` [validate-redirects run](https://github.com/mdn/translated-content/runs/6740649893?check_suite_focus=true)

@cw118 thank you for the heads up! :) I've resolved the remaining conflicts and this branch is now up to date with `main`, therefore the `fr`, `zh-CN`, and `zh-TW` incoming changes have been accepted/used on this branch overriding my previous changes. Let me know if I can do anything in the meantime.

In terms of removing all of the changes from only those folders, it wasn't so straightforward.  I thought about a `git revert` or `git reset` or rebase but the fr/zh-CN/zh-TW changes were not in specific commits and were apart of other commits that changed other directories, so "going back in git time" would cause more problems than I know how to handle. If someone has a better understanding/process for removing all changes from fr/zh-CH/zh-TW, feel free to direct me!

All in all, this PR became too much to handle and should have broken into localized PRs from the start. But I think from the discussion above thats clear for this types of PRs moving forward.

**Comment by schalkneethling** [comment]
@fiji-flo could you have a look at why this test is failing? Thank you.

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
Regarding [files/fr/web/css/cross-fade()/index.md](https://github.com/mdn/translated-content/pull/3085/files#diff-92399794fdd7e2d2a07bcc513c2f3026a3af7079a1524905ba92b9c030bb689b), the page is already present in French: 
https://github.com/mdn/translated-content/blob/main/files/fr/web/css/cross-fade/index.md

I think it should not be added by this PR.

**Comment by tannerdolby** [comment]
@SphinxKnight Good catch! I've removed `files/fr/web/css/cross-fade()/index.md` from this PR as the page already existed like you mentioned at `files/fr/web/css/cross-fade/index.md`.

Tests seem to be all passing now!

**Comment by irvin** [comment]
@tannerdolby we need to remove several additional html files in /ja and one in /ko (that could had be converted to md already)

**Review by irvin — CHANGES_REQUESTED** [review]
截圖 2022-06-14 下午10 33 15

these files need to be removed before we can merge.

**Comment by schalkneethling** [comment]
> @irvin thanks for pinpointing the files! I will handle those shortly and update this branch.

Thank so much for your hard work on this one Tanner, we really appreciate it.

**Comment by tannerdolby** [comment]
@schalkneethling No problem at all :) Thanks to everyone who participated in this PR! 

I know it was a lot of cleaning up merge conflicts, but I appreciate people fixing things where they could and pointing me in the right direction for this branch. Hopefully we can land this PR soon!

**Pull request merged**